tests/transforms/test_zx.py

```diff
@@ -0,0 +1,654 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the `pennylane.transforms.zx` folder.
+"""
+import sys
+
+import numpy as np
+import pytest
+import pennylane as qml
+from pennylane.tape import QuantumScript
+
+pyzx = pytest.importorskip("pyzx")
+
+pytestmark = pytest.mark.zx
+
+supported_operations = [
```

Comment:
will the list of operations supported by ZX calculus ever change? updating this list might fall through the cracks when there's also a list in the source code.

Reply:
Yes they might have some changes and additions, but I prefer having some control. The problem is that we do not want to import pyzx in PennyLane then the supported list is buried inside the `to_zx` function.

Reply:
ahh because the values in the map in `to_zx` need pyzx! no problem